go.mod

```diff
@@ -62,6 +55,4 @@ require (
 
 replace github.com/kilic/bls12-381 => github.com/kilic/bls12-381 v0.0.0-20200820230200-6b2c19996391
 
-replace github.com/ethereum/go-ethereum => github.com/textileio/go-ethereum v1.10.3-0.20210413172519-62e8b38d82b1
```

Comment:
I dropped this replacement, but let me know if this was really needed for something.

Reply:
No these are probably leftovers when extracting this out.